### DISCUSSION TITLE
Doris字符串字段，使用string或者varchar类型有性能区别吗？

### DISCUSSION THREAD

**Discussion by a3225167** [apache/doris#15733] [A - General / Q&A]
1）由于之前使用的是hbase，不需要关心字段的长度， 现在换成Doris后 字符串的长度 不知道要定义成多少，可以全部用string吗？
2）全部使用string类型对性能有影像吗？ 还是一定要用varcahr？

**Comment by linzch3** [comment]
I have the same question

**Comment by yuanyuan8983** [comment]
It mainly depends on the demand. If there is a long string demand, you can use string. If the string is not very long, you can use varchar type. The specific business needs, you can add me WeChat：cyllyy810222
